securesignals-ima-devapp/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,16 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <!-- Base application theme. -->
+    <style name="Theme.UID2AndroidSDK" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
```

Comment:
Let's remove the custom theme, I don't think it's required.

Reply:
This is needed to power the AppCompatActivity.

Reply:
Done.